defaults/jq/semconv.jq

```diff
@@ -0,0 +1,102 @@
+def semconv_group_attributes_by_namespace:
+    group_by(.namespace)
+    | map({ namespace: .[0].namespace, attributes: . | sort_by(.name) });
+
+#####################
+# Attribute functions
+#####################
+def semconv_attributes($options):
+    .groups
+    | map(select(.type == "attribute_group" and (.id | startswith("registry."))))
+    | map(.attributes) | add
+    | if ($options | has("exclude_stability")) then
+        map(select(.stability as $st | $options.exclude_stability | index($st) | not))
+      else
+        .
+      end
+    | if ($options | has("exclude_deprecated") and $options.exclude_deprecated == true) then
+        map(select(has("deprecated") | not))
+      else
+        .
+      end
+    | map(. + {namespace: (if .name | index(".") then .name | split(".")[0] else "other" end)})
+    | if ($options | has("exclude_namespace")) then
+        map(select(.namespace as $st | $options.exclude_namespace | index($st) | not))
+      else
+        .
+      end
+    | sort_by(.namespace, .name);
+
+def semconv_attributes: semconv_attributes({});
+
+def semconv_grouped_attributes($options):
+    semconv_attributes($options)
+    | semconv_group_attributes_by_namespace;
+
+def semconv_grouped_attributes: semconv_grouped_attributes({});
+
+# Generic Signal Functions
+def semconv_group_signals_by_namespace($signal):
+    group_by(.namespace)
+    | map({ namespace: .[0].namespace, ($signal): . | sort_by(.name) });
+
+def semconv_signal($signal; $options):
+    .groups
+    | map(select(.type == $signal))
+    | if ($options | has("exclude_stability")) then
+        map(select(.stability as $st | $options.exclude_stability | index($st) | not))
+      else
+        .
+      end
+    | if ($options | has("exclude_deprecated") and $options.exclude_deprecated == true) then
+        map(select(.id | endswith(".deprecated") | not))
+      else
+        .
+      end
+    | map(. + {namespace: (if .id | index(".") then .id | split(".") | .[1] else "other" end)})
```

Comment:
the group id is not a namespace, at least not today. I don't think we have a way to group span, resources, or scopes (also there are no scopes in the schema anymore). 

Events have a name, but I don't think we want anyone to codegen things for span events. And new log-based events are not there yet. 

So I suggest removing support for resource, scopes, spans, and events in another comment and we can revisit it once we start doing codegen for them.

Reply:
For now, I don’t have a problem with that. @jsuereth, are you okay with that as well?

Reply:
Absolutely. I don't want to commit to supporting concepts we haven't evolved enough in semconv (that's related to my earlier comments).

Let's keep this focused and expand as we know more.

Reply:
Done